### PR TITLE
[CELEBORN-772] Convert StreamChunkSlice, ChunkFetchRequest, TransportableError to PB

### PR DESCRIPTION
**Pull request by SteNicholas**
### What changes were proposed in this pull request?

`StreamChunkSlice`, `ChunkFetchRequest` and `TransportableError` should merge to transport messages to enhance celeborn's compatibility.

### Why are the changes needed?

1. Improves celeborn's transport flexibility to change RPC.
2. Makes Compatible with 0.2 client.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

- `FetchHandlerSuiteJ`
- `RequestTimeoutIntegrationSuiteJ`
- `ChunkFetchIntegrationSuiteJ`

### CODE REVIEW

**Comment by SteNicholas** [comment]
@RexXiong, @FMX, could you help to review this pull request?

File: common/src/main/proto/TransportMessages.proto [hunk 1]
```diff
@@ -551,3 +554,19 @@ message PbReadAddCredit {
...
+  int32 len = 4;
+}
+
+message PbChunkFetchRequest {
+  PbStreamChunkSlice streamChunkSlice = 1;
+}
+
+message PbTransportableError {
+  int64 streamId = 1;
+  string errorMessage = 2;
```
Comment (RexXiong) [inline common/src/main/proto/TransportMessages.proto:571]: LGTM, exception a nit: errorMessage -> message (message of TransportableError has the `error` meaning).

**Review by RexXiong — APPROVED** [review]
LGTM, thanks!

**Comment by RexXiong** [comment]
Thanks, merge to main

**Pull request closed without merging**